test/test_onnx.py

```diff
@@ -123,9 +135,9 @@ def __init__(self_module):
             def forward(self_module, images):
                 return self_module.transform(images)[0].tensors
 
-        input = [torch.rand(3, 100, 200), torch.rand(3, 200, 200)]
-        input_test = [torch.rand(3, 100, 200), torch.rand(3, 200, 200)]
-        self.run_model(TransformModule(), [input, input_test])
+        input = torch.rand(3, 100, 200), torch.rand(3, 200, 200)
+        input_test = torch.rand(3, 100, 200), torch.rand(3, 200, 200)
+        self.run_model(TransformModule(), [(input,), (input_test,)])
```

Comment:
hum, interesting, was this a bug in the testing before?

Reply:
This wasn't a bug, I just refactored it to be inconsistent with other tests